app/controllers/haikus_controller.rb

```diff
@@ -13,20 +13,18 @@ def show
   def create
     haiku = Haiku.new(haiku_params)
     if haiku.save
-      render json: haiku
-      head 201
+      render json: haiku.lines
```

Comment:
I'm not sure if I'm understanding correctly. I don't think we need the 201 there, so I removed it. The status would be 200 now and json will show the haiku lines for the haiku created.